### ISSUE TITLE
Webview not work after gui.Window.open() [$300]

### ISSUE THREAD

**Issue by bachel** [nwjs/nw.js#3302]
Webview is loading pages only if webview element is in main page (page specified in package.json as "main")

Here is example code:
Page "https://google.com" is never loaded, but "https://github.com" load as expected.
package.json
```javascript
{
  "name": "TEST",
  "main": "app://test/init.html"
}
```
init.html
```javascript
<html>
    <webview src="https://github.com"/>
    <script>
         var gui = require("nw.gui");
         gui.Window.open("web.html")
     </script>
</html>
```

web.html
```javascript
<html>
     <webview src="https://google.com"/>
</html>
```

**Comment by bachel** [comment]
simillar to #3188

**Comment by justinmchase** [comment]
This might be a dumb question, but  you are missing `<body>` tags. Does putting it in body tag make a difference?

**Comment by bachel** [comment]
No, there is no difference :-) I removed body tags because I want to simplify the example.

**Comment by kamesh-a** [comment]
But, creating a webview dom element, from mian container window and appending the dom, in child window from main works fine.

ex: 
mainWindow : creating "webview" element with any source, src = 'http://google.com';
childWindow : created with gui.Window.open();
mainWindow : childWindow.appWindow.window.document.body.append(<webviewDOMweCreated>);

Works in child. just in case, some one like to use the work around or just to it know this ways gives patch to just work with, until the global Fix comes in.

**Comment by bachel** [comment]
Great ! Works for me.

**Comment by bachel** [comment]
Fixed in 0.13.0 alfa2
